scripts/create_tables.sql

```diff
@@ -1 +1,34 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+
+-- create a table to store information about patients
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT,
+  insurance TEXT
+);
+
+-- create a table to store information about patients
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  appt_date DATE,
+  appt_time TIME,
+  comment TEXT,
+  patient_id INTEGER REFERENCES patients(id),
+  doctor_id INTEGER REFERENCES doctors(id)
+);
+
+CREATE INDEX ON appointments(patient_id);
+CREATE INDEX ON appointments(doctor_id);
```

Comment:
Good, we don't need the CREATE INDEX line at the end since we used the REFERENCES already earlier.